src/Message/SNS/Envelope.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Kfz24\QueueBundle\Message\SNS;
+
+use JMS\Serializer\Annotation as Serializer;
+
+/**
+ * Class Envelope
+ * @package Kfz24\QueueBundle\Message\SNS
+ */
+class Envelope
+{
+    /**
+     * @var MessageInterface
+     * @Serializer\MaxDepth(1)
+     */
+    protected $message;
+
+    /**
+     * @var \DateTimeImmutable
+     * @Serializer\Type("DateTimeImmutable")
+     */
+    protected $createdAt;
+
+    /**
+     * AbstractEnvelop constructor.
```

Comment:
Same about this comment? Just noise to me.

Reply:
If we add a code style/static checker, I think it will throw an error if the doc block is missing for any method.

Reply:
I think he's referring to the `AbstractEnvelop constructor.` flavor text, not the param line underneath :)

That one is definitely not needed from the checker (source: I had the same line from PHP storm in one of my PRs and Nick came and made me remove it :smile: )

Reply:
Ah, ok. I removed it